### PR TITLE
Chapter4 code

### PR DESCRIPTION
**Pull request by mmargenot**

### CODE REVIEW

File: Chapter4_TheGreatestTheoremNeverTold/top_crazyideas_submissions.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    try:
+        ratio = reddit.get_submission(sub.permalink).upvote_ratio
+        ups = int(round((ratio*sub.score)/(2*ratio - 1)) if ratio != 0.5 else round(sub.score/2))
+        upvotes.append(ups)
+        downvotes.append(ups - sub.score)
+        contents.append(sub.title)
+    except Exception as e:
+        continue
+votes = np.array( [ upvotes, downvotes] ).T
+
```
Comment (twiecki) [inline Chapter4_TheGreatestTheoremNeverTold/top_crazyideas_submissions.py]: remove trailing spaces

Comment (mmargenot) [inline Chapter4_TheGreatestTheoremNeverTold/top_crazyideas_submissions.py]: Just removed them

**Comment by twiecki** [comment]
Why did you comment out the find_MAP() call? I'd either remove it or use it.

**Comment by twiecki** [comment]
Otherwise LGTM :+1:

**Comment by mmargenot** [comment]
It was actually the source of a strange bug I was getting which I didn't particularly like. I'll remove it.

**Pull request merged**